app/swagger/swagger/requests/profile.rb

```diff
@@ -1415,6 +1415,113 @@ class Profile
           end
         end
       end
+
+      swagger_path '/v0/profile/contacts' do
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+          extend Swagger::Responses::ForbiddenError
+
+          key :description, 'Create a Next-of-Kin or Emergency Contact'
+          key :tags, [:profile]
+
+          parameter :authorization
+
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Contact attributes'
+            key :required, true
+
+            schema do
+              key :$ref, :Contacts
+            end
+          end
+
+          response 201 do
+            key :description, 'Contact created'
+          end
+
+          response 422 do
+            key :description, 'Unprocessable entity'
+            schema do
+              key :$ref, :Errors
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/profile/contacts/{id}' do
+        operation :patch do
```

Comment:
I am not familiar with the use of path in the APIs, but should this be update?

Reply:
Yes. A HTTP `PUT` or `PATCH` should translate to an update operation (in this case, the `#update` method of the `ContactsController`)

Reply:
I'm expecting the front-end to send a completed form, so I'll swap `PATCH` for `PUT`, here.